recipes/mdtraj/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "mdtraj" %}
+{% set version = "1.7.2" %}
+{% set sha256 = "1bc9fb06e84f6562e51e040946dae0b940eaca4760f78f3ac451f4a89b132a9a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  entry_points:
+    - mdconvert = mdtraj.scripts.mdconvert:entry_point
+    - mdinspect = mdtraj.scripts.mdinspect:entry_point
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - cython
+    - numpy x.x
+
+  run:
+    - python
+    - setuptools
```

Comment:
@msarahan there are `pkg_resources` imports all over the code. So I guess it is...